Week2/Homework/execute-queries.js

```diff
@@ -0,0 +1,127 @@
+const mysql = require('mysql');
+const util = require('util');
+const prompt = require('prompt');
+
+const connection = mysql.createConnection({
```

Comment:
Similar to the other file, `execute-alert.js`, you're again creating a connection to the database here. A better practice would be to put this into a separate file and then load it into both files. We want to do this so that each file only has a single purpose (and is therefore easier to read and understand).

Reply:
Thanks... I've made a separate file and loaded it into both files